### ISSUE TITLE
Pagination

### ISSUE THREAD

**Issue by brianteeman** [joomla/jed-issues#224]
Select the "Compatible with Joomla 4 Beta" menu item and try to navigate to page 2, 3 etc

The first pagination works but after that you are just returned to the home page

**Comment by himanshu007-creator** [comment]
> Select the "Compatible with Joomla 4 Beta" menu item and try to navigate to page 2, 3 etc
> 
> The first pagination works but after that you are just returned to the home page

Yes @brianteeman , i was going to open same issue, i was looking for an extension that require a download key to update for 
<code><a href="https://github.com/joomla/joomla-cms/issues/33698">this</a></code> issue. Please suggest such extension.
Thanks.

**Comment by anibalsanchez** [comment]
Hi,

The plan is to remove this menu item once J4 is released. It was published at the time Joomla published the initial J4 Alpha and Beta versions as a temporary way to give visibility to the new extensions for Joomla 4.

Now, the list of J4-compatible extensions has 39 pages, so it makes no sense to keep it and the whole directory is moving to support extensions for J3 & J4.

What do you think @Llewellynvdm? Do we close the issue and unpublish the **Compatible with J4 Beta** now? Or, do we wait for the release of Joomla 4 RC?
